exporter.go

```diff
@@ -157,6 +166,53 @@ func (e *Exporter) Export(opts ExportOptions) (platform.ExportReport, error) {
 	return report, nil
 }
 
+func copySboms(layerDir string, e *Exporter) error {
+	targetBuildDir := filepath.Join(layerDir, "sbom", "build", "buildpacksio_lifecycle")
+	targetLaunchDir := filepath.Join(layerDir, "sbom", "launch", "buildpacksio_lifecycle")
+
+	if err := os.MkdirAll(targetBuildDir, os.ModePerm); err != nil {
+		return errors.Wrapf(err, "fail to create directory %v", targetBuildDir)
+	}
+
+	if err := os.MkdirAll(targetLaunchDir, os.ModePerm); err != nil {
+		return errors.Wrapf(err, "fail to create directory %v", targetLaunchDir)
+	}
+
+	extensions := [3]string{".sbom.cdx.json", ".sbom.spdx.json", ".sbom.syft.json"}
+	components := [2]string{"lifecycle", "launcher"}
+
+	var targetDir string
+	for _, component := range components {
+		if component == "lifecycle" {
+			targetDir = targetBuildDir
+		} else if component == "launcher" {
+			targetDir = targetLaunchDir
+		}
+		for _, extension := range extensions {
+			sbomFilename := component + extension
+
+			srcSbomAbsPath := "/cnb/lifecycle/" + sbomFilename
```

Comment:
We should use `filepath.Join` and on Windows, we'll need to add the root directory (we do something similar here: https://github.com/buildpacks/lifecycle/blob/5a4df34080c097528eca957576ae19e477942398/platform/defaults_windows.go#L5)

Reply:
ok. I have pushed a new commit to fix this problem:
https://github.com/omocquais-p/lifecycle/commit/f230fdeffa5a40b7e0b0bc1784af7562248f33db